edb/schema/types.py

```diff
@@ -1868,7 +1868,7 @@ def as_colltype_delete_delta(
                 expiring_refs=expiring_refs,
             )
 
-        for el in self.get_subtypes(schema):
+        for el in set(self.get_subtypes(schema)):
```

Comment:
What's this for?

Reply:
Don't delete nested types twice if they appear twice

I'll add a comment

Reply:
Actually that wasn't really sufficient; I switched to using if_exists